vktrace/vktrace_replay/vkreplay_main.cpp

```diff
@@ -341,6 +348,18 @@ int vkreplay_main(int argc, char** argv, vktrace_window_handle window = 0) {
         vktrace_set_global_var("VK_SCREENSHOT_FRAMES", "");
     }
 
+    // Set up environment for screenshot color space format
+    if (replaySettings.screenshotColorFormat != NULL && replaySettings.screenshotList != NULL) {
+        vktrace_set_global_var("VK_SCREENSHOT_FORMAT", replaySettings.screenshotColorFormat);
+        vktrace_free((char*)replaySettings.screenshotColorFormat);
+    }else if (replaySettings.screenshotColorFormat != NULL && replaySettings.screenshotList == NULL) {
+        vktrace_LogError("Screenshot format should be used when screenshot enabled!");
```

Comment:
Maybe make this a warning: "Screenshot format ignored when screenshot is not enabled"